project-base/storefront/components/Forms/Button/Button.tsx

```diff
@@ -52,8 +52,8 @@ export const getButtonClassName = (
     return twJoin(
         'inline-flex w-auto h-fit cursor-pointer items-center justify-center gap-2 rounded-lg text-center font-bold outline-none transition-all hover:no-underline',
         'outline-2 outline-offset-[-2px]',
-        size === 'small' && 'px-3 py-2 text-xs sm:py-1 sm:px-4 sm:text-sm',
-        size === 'medium' && 'py-3 px-8',
+        size === 'small' && 'px-4 py-[6px] text-xs',
```

Comment:
We can utilize Tailwind's class for this `py-1.5` - note that for small values (up to 14px) the span is
- 0 -> 0px
- 0.5 -> 2px
- 1 -> 4px
- ...
- 3.5 -> 14px

Reply:
Buttons will be different anyway. Already working on Typography and Buttons in different task :)